packages/mantle/src/components/accordion/accordion.tsx

```diff
@@ -35,7 +35,7 @@ AccordionTrigger.displayName = "AccordionTrigger";
 const AccordionTriggerIcon = ({ className, ...props }: Omit<IconProps, "svg">) => (
 	<Icon
 		{...props}
-		svg={<CaretDown weight="fill" />}
```

Comment:
<img width="280" alt="image" src="https://github.com/user-attachments/assets/6f6f917f-00ba-4879-aaca-37f2c9556f52" />

I think this treatment is just too thin and disappear in this context (at least). These also need to be updated with the carets as the leftmost item. Also wondering if we should just allow the user to choose which icon weight for these depending on the context.

Could we just remove this part from this PR?

Reply:
yeah, the accordion component is getting a makeover soon and there's gonna be multiple variants